apps/pie-docs/src/engineers/contributing.md

```diff
@@ -8,4 +8,38 @@ title: Contributing
 description: In this section you’ll find everything you need to know about contributing to our design system and how you can help us.
 ---
 
-## Introduction
+### How can I get in contact with the Design System team?
+
+You can reach out to us on one of our Slack channels (JET employees only):
+- **#help-designsystem**
+- **#guild-designsystem**
+- **#guild-frontend**
+- **#guild-accessibility**
+
+We also run a [PIE engineering clinic](https://calendly.com/jetpie/engineering-clinics) every other Tuesday where you can book a slot to speak to one of our engineers.
+
+{% notification {
+type: "information",
+message: "Note that the booking link only goes live the day before the clinic."
+} %}
+
+### How can I make a feature request?
+
+Please use one of the workflows in the **#help-designsystem** channel to make a feature request, report a defect or request help for anything else.
+
+### How can I find out which components have already been built?
+
+You can take a look at our Storybook instances:
+- [Vue](https://vue.pie.design/)
+- [React for Takeaway.com](https://snacks.takeaway.com/)
+- [React for SkipTheDishes](https://react.pie.design/)
+
+### Can I build a new component?
+
+Yes. If the component has been approved by designers then you are more than welcome to contribute.
+
+Please speak to the Design System team about your requirements and we will help by guiding you.
+
+### Can I make a custom implementation for a component that already exists?
+
+No. We suggest that you raise a feature request describing your needs and we will discuss how they can be met by using (or extending) an existing component.
```

Comment:
this could be a fun opportunity to use the warning or error notifications :)

Reply:
Could be fun... At the notification semantically is an `aside`, i.e., not directly related to the main content, whereas the answer is directly relevant, so it might not work as-is